### ISSUE TITLE
[Accessibility]: Difficult to jump into code view after selecting the file from "Files" in sidebar

### ISSUE THREAD

**Issue by Numbers88s** [sourcegraph/sourcegraph#35136]
### Audit type

Keyboard navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/33507

### Problem description
<img width="1376" alt="finding markup" src="https://user-images.githubusercontent.com/6225160/167516203-401085d8-fe61-4072-ba4e-fc349e9e4205.png">

When selecting the desired directory it is quite easy to open the file but difficult to jump to it and read the code. ❌ 

- Navigate to a repository in Sourcegraph
- Select “Files” in the sidebar if it is not selected.
- Click on directories to drill down further, potentially scrolling to navigate to the desired directory / file.
- Click on a file name to open it 
- **_Try to jump into the code to read it_** 😞 

### Expected behavior

I would expect to tab or select the file and then be able to move to the file. I could not find a way to do this atm. 😢 

### Assigning labels

- [x] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [x] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [x] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [x] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

I'm unsure who should work on this issue or if it needs more clarification. I will let the Frontend Platform team triage this.

**Comment by umpox** [comment]
Duplicate of https://github.com/sourcegraph/sourcegraph/issues/35072, closing this!
